src/kpi/dlt-kpi-options.c

```diff
@@ -136,20 +136,16 @@ DltReturnValue dlt_kpi_read_configuration_file(DltKpiConfig *config, char *file_
         return DLT_RETURN_ERROR;
     }
 
-    if((line = malloc(COMMAND_LINE_SIZE)) == 0)
-    {
-        fprintf(stderr, "%s: Out of memory!\n", __func__);
-        return DLT_RETURN_ERROR;
-    }
+    if(((line = malloc(COMMAND_LINE_SIZE)) == 0) ||
+       ((token = malloc(COMMAND_LINE_SIZE)) == 0) ||
+       ((value = malloc(COMMAND_LINE_SIZE)) == 0))
 
-    if((token = malloc(COMMAND_LINE_SIZE)) == 0)
     {
-        fprintf(stderr, "%s: Out of memory!\n", __func__);
-        return DLT_RETURN_ERROR;
-    }
+        fclose(file);
```

Comment:
initialize these pointers while declaring in order to avoid possible free of uninitialized pointer

Reply:
Thanks for the suggestion! The commit has been updated.